app/src/Process/Process.js

```diff
@@ -0,0 +1,183 @@
+import Footer from "../Components/Footer/Footer";
+import Navbar from "../Components/Navbar/Navbar";
+import { DotLottieReact } from "@lottiefiles/dotlottie-react";
+import { MyCodeBlock } from "../Components/MyCodeBlock/MyCodeBlock";
+import { SiKakaotalk } from "react-icons/si";
+import { FaMagnifyingGlass } from "react-icons/fa6";
+import { FaPencilAlt } from "react-icons/fa";
+import { HiMiniPresentationChartBar } from "react-icons/hi2";
+import { HiMiniRocketLaunch } from "react-icons/hi2";
+
+import 'react-vertical-timeline-component/style.min.css';
+import {
+  VerticalTimeline,
+  VerticalTimelineElement,
+} from "react-vertical-timeline-component";
+
+const code = `import React from 'react';
+import ReactDOM from 'react-dom/client';
+import { BrowserRouter, Routes, Route } from "react-router";
+import './index.css';
+import reportWebVitals from './reportWebVitals';
+import Home from './Home/Home';
+import { Process } from './Process/Process';
+
+const root = ReactDOM.createRoot(document.getElementById('root'));
+root.render(
+    <React.StrictMode>
+        <BrowserRouter>
+            <Routes>
+                <Route path='/' element={<Home/>}></Route>
+                <Route path='/process' element={<Process/>}></Route>
+            </Routes>
+        </BrowserRouter>
+    </React.StrictMode>
+);`;
```

Comment:
...
No, just no.

Instead of passing it as a string, you can pass a function instead. That would be safer and would adhere to standards.